pymc/tests/distributions/test_continuous.py

```diff
@@ -373,6 +375,12 @@ def test_beta_logp(self):
             {"mu": Unit, "sigma": Rplus},
             beta_mu_sigma,
         )
+        check_logp(
```

Comment:
No need to test the logp (which is a heavy test). Should be enough to add a test like this: https://github.com/pymc-devs/pymc/blob/3ff4e7af14afbb5f3221572657e04f97e1ae4b44/pymc/tests/distributions/test_continuous.py#L2013-L2020

Reply:
Thank you for the example, I added the equivalent test and removed the logp test.